samples/exposed-spring/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+val exposedVersion: String by project
+
+plugins {
+    id("org.springframework.boot") version "3.1.2"
+    id("io.spring.dependency-management") version "1.1.2"
+    kotlin("jvm") version "1.8.21"
+    kotlin("plugin.spring") version "1.8.21"
+}
+
+group = "org.jetbrains.exposed"
+version = "1.0.0"
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
+}
+
+kotlin {
+    jvmToolchain(17)
+}
```

Comment:
@e5l I applied jvmtoolchain. Are you sure it applied as you intended?

+Cannot remove freeCompilerArgs field that jvmtoolchain could not receive freeCompilerArgs. If you know how to solve it, please advice to me

Reply:
@FullOfOrange I ran the sample locally using the updated build file (with jvmToolchain in `kotlin` block and freeCompilerArgs in the original `kotlinOptions` block). And it works without any issue for me.

Is running the sample causing a compilation issue on your end? Or you'd like confirmation that everything looks good with the build setup?

Reply:
No issues for me either. I asked the question because this is the first time I've seen jvmToolchain, so I thought maybe I could replace freeCompilerArgs with jvmToolchain.

Reply:
Existing code is generated by the spring initializer, it seems good to use the default template code without using jvmtoolchain.

Reply:
I think it's better to add it, the wizard is not always up to date and with `jvmToolchain` we will have less errors (or better error message) about jdk version